eng/Packaging.props

```diff
@@ -12,6 +12,10 @@
     <PackagePlatform Condition="'$(PackagePlatform)' == 'amd64'">x64</PackagePlatform>
     <NativePackagePath>$(MSBuildThisFileDirectory)src/Native/pkg</NativePackagePath>
 
+    <XmlDocPackage>Microsoft.Private.Intellisense</XmlDocPackage>
```

Comment:
Should this instead be in Version.props?  We don't typically put versions in random places in the repo.  Optimistically we'll one day ingest this through maestro as well.

Reply:
Actually I think the only thing you need is XmlDocFileRoot.  See https://github.com/dotnet/arcade/blob/247c0290c554a68dd3e560dd88fb0da26353d9d7/src/Microsoft.DotNet.Build.Tasks.Packaging/src/build/PackageLibs.targets#L54


That can be defined using the properties that Viktor moved 
https://github.com/dotnet/corefx/commit/c4c1985531cc1f25c805bbc4801b85c98e0dfee0#diff-4e8b29963372e731e4662efaf22b414dR12

Alternatively we could change the packaging targets to use the dirs that @ViktorHofer’s change copied to.

Reply:
I missed that `XmlDocFileRoot` is used in `PackageLibs.targets`, we should a comment to point that out.